aws-ts-serverless-raw/index.ts

```diff
@@ -27,31 +27,44 @@ let counterTable = new aws.dynamodb.Table("counterTable", {
 // Lambda Function
 ///////////////////
 
-// Create a Role giving our Lambda access.
-let policy: aws.iam.PolicyDocument = {
-    "Version": "2012-10-17",
-    "Statement": [
-        {
-            "Action": "sts:AssumeRole",
-            "Principal": {
-                "Service": "lambda.amazonaws.com",
-            },
-            "Effect": "Allow",
-            "Sid": "",
-        },
-    ],
-};
-let role = new aws.iam.Role("mylambda-role", {
-    assumeRolePolicy: JSON.stringify(policy),
+// Give our Lambda access to the Dynamo DB table, CloudWatch Logs and Metrics.
+const role = new aws.iam.Role("mylambda-role", {
+    assumeRolePolicy: pulumi.output(
```

Comment:
While we are changing this - we could simplify further with https://pulumi.io/reference/pkg/nodejs/pulumi/aws/iam/#assumeRolePolicyForPrincipal.